# ClusterInfo.msg
int32 label
float32[] centroid
int32 size

